package/yast2-crowbar.changes

```diff
@@ -72,7 +78,7 @@ Wed Oct  7 13:52:52 CEST 2015 - jsuchome@suse.com
 
 - Adapt module so it can be called during first stage of installation
   (which is the only stage in SLES12)
-- 3.1.8 
+- 3.1.8
```

Comment:
yeah, my linter is cleaning up the trailing spaces, but I believe that is fine, right?